rich-messages-code-sample/Dialogflow CX Webhook

```diff
@@ -0,0 +1,64 @@
+Dialogflow CX Webhook
+
+Using webhooks in Dialogflow CX is easy. 
+
+1. Enable the webhook for your intent. If you have created a webhook already, you can select the webhook you want to use or create a new one.
+
+2. You can also create the webhooks under Manage Section.
+
+Once you have enabled the webhook and set it up your response. It will be rendered on Kommunicate UI:
+
+Here is an example of a webhook response for buttons and links buttons:
```

Comment:
in this repo , we have to add sample code to send rich msgs via webhook. which customer can just run and try with their df cx bot. 
this written description is already there in docs.

Reply:
like this
https://github.com/Kommunicate-io/Kommunicate-Web-SDK/tree/8c2d6322e19c5bc7734a2ca973db395a18fe60b3/rich-messages-code-sample/dialogflow-fulfillment

Reply:
added all in single file see if this works for you